apps/frontend/src/modules/quiz/slices/quiz.slice.ts

```diff
@@ -6,22 +6,27 @@ import {
 } from "~/libs/constants/constants.js";
 import { DataStatus } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
-import { type QuizQuestionDto } from "~/modules/quiz/quiz.js";
+import {
+	type QuizQuestionDto,
+	type QuizScoresGetAllItemResponseDto,
+} from "~/modules/quiz/quiz.js";
 
-import { getAllQuestions } from "./actions.js";
+import { getAllQuestions, getScores } from "./actions.js";
 
 type State = {
 	currentCategory: null | QuizQuestionDto[];
 	currentCategoryIndex: number;
 	dataStatus: ValueOf<typeof DataStatus>;
 	questions: QuizQuestionDto[][];
+	scores: [] | QuizScoresGetAllItemResponseDto[];
```

Comment:
```suggestion
	scores: QuizScoresGetAllItemResponseDto[];
```

Reply:
If I am not mistaken, this way emty array also will be allowed

Reply:
It works great. Got rid of unnecessary empty array